Gems/Maestro/Code/Source/Cinematics/2DSpline.h

```diff
@@ -35,9 +32,20 @@ namespace spline
         float theta_from_dd_to_ds;
         float scale_from_dd_to_ds;
 
-        void ComputeThetaAndScale() { assert(0); }
-        void SetOutTangentFromIn() { assert(0); }
-        void SetInTangentFromOut() { assert(0); }
+        void ComputeThetaAndScale()
+        {
+            AZ_Assert(false, "Not expected to be used");
+        }
+
+        void SetOutTangentFromIn()
+        {
+            AZ_Assert(false, "Not expected to be used");
+        }
+
+        void SetInTangentFromOut()
+        {
+            AZ_Assert(false, "Not expected to be used");
+        }
```

Comment:
odd, why are these implemented if they're not expected to be used? can they be removed?